### PR TITLE
WACOP: style updates, map fix, and empty address handling

### PR DESCRIPTION
**Pull request by dhornbein**
moved read more on event box

Fixed map on home page

removed event/alert boxes

Conditional address data on incident profile

Missing address handling for alert and event html

Tagit visual improvement

Removed list style from update comments

### CODE REVIEW

File: modules/templates/WACOP/controllers.py [hunk 1]
```diff
@@ -683,7 +681,7 @@ def _events_html():
                     # No location
                     L1_abrv = ""
                     L3 = ""
-                    addr_street = ""
+                    addr_street = "No Address Given"
```
Comment (flavour) [inline modules/templates/WACOP/controllers.py]: I'll wrap this in T()

File: modules/templates/WACOP/views/incident_browse.html [hunk 2]
```diff
@@ -1,8 +1,6 @@
 {{extend "layout.html"}}
 {{=system_wide}}
 <main role='main' class='main'>
- {{=XML(alerts)}}
- {{=XML(events)}}
```
Comment (flavour) [inline modules/templates/WACOP/views/incident_browse.html]: if we are removing these from the view, then we should also not bother generating them => saves on server load!

Comment (dhornbein) [reply modules/templates/WACOP/views/incident_browse.html]: agreed, what/where does that happen? We still need them for other pages.

File: modules/templates/WACOP/controllers.py (hunk 1 above)
Comment (dhornbein) [reply modules/templates/WACOP/controllers.py]: thanks! I take it T() does the translation? I'll keep note of that for the future.

File: modules/templates/WACOP/views/incident_browse.html (hunk 2 above)
Comment (flavour) [reply modules/templates/WACOP/views/incident_browse.html]: > what/where does that happen?

See my followup commit: https://github.com/sahana/eden/commit/244d7a185019942cb4d16c6f0ac6d4a11b847bc6

> I take it T() does the translation?

Yes.

**Pull request merged**